domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -62,6 +63,7 @@
     private final LocalDate moneyReceivedOn;
     private final ReDetermination reDetermination;
     private final LocalDateTime reDeterminationRequestedAt;
+    private final ClaimDocumentStore claimDocumentStore;
```

Comment:
it should be store, can Documents or in similarity to ccd ClaimDocuments

Reply:
The reason I chose to call it store is coz the class holds a collection of documents. I didn't want to use Collection because that may be confused with java collection.